test/test_helper.exs

```diff
@@ -1 +1,3 @@
 ExUnit.start()
+Mox.defmock(Translixir.MockHTTPoison, for: HTTPoison.Base)
+HTTPoison.start()
```

Comment:
Probably this is the fix to that previous error that you showed me up, right?

Reply:
Yes kkkkk